phys/module_physics_init.F

```diff
@@ -45,6 +45,7 @@ SUBROUTINE phy_init ( id, config_flags, DT, restart, zfull, zhalf,     &
                          STEPBL,STEPRA,STEPCU,                   &
                          W0AVG, RAINNC, RAINC, RAINCV, RAINNCV,  &
                          SNOWNC, SNOWNCV, GRAUPELNC, GRAUPELNCV, &
+                         lpos,lneg,lneu, &
```

Comment:
move this line to later in the argument list to make it optional. Safest is near the end.